### PR TITLE
ENDOC-576 7.1 check docs for needed changes wrt bundle-id, plugin-id

### PR DESCRIPTION
**Pull request by Lyd1aCla1r3**

### CODE REVIEW

File: vuepress/docs/next/tutorials/devops/plugin-configuration.md [hunk 1]
```diff
@@ -28,39 +28,28 @@ resources.limits.memory:    integer, mebibytes
...
-If you just have a bundle, use this command with the appropriate parameters
+Use the following command from the root bundle project directory:
 ```sh
-ent ecr get-plugin-id --autho --repo=YOUR-BUNDLE-REPO-URL
-```
-Example:
-```sh
-$ ent ecr get-plugin-id --auto --repo=https://github.com/entando-samples/entando-hub-application-bundle.git
-pn-cee95efc-77ff566e-entandopsdh-entando-hub-catalog-ms
+ent ecr get-plugin-code
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/devops/plugin-configuration.md:38]: I could only get this to work with ent ecr get-plugin-code [component-name] --repo=[full-repository-url docker://registry...]

File: vuepress/docs/next/tutorials/devops/plugin-configuration.md [hunk 2]
```diff
@@ -28,39 +28,28 @@ resources.limits.memory:    integer, mebibytes
...
 ```yaml
 data:
-  entando.profile.plugins.YOUR-PLUGIN-ID: |-
+  entando.profile.plugins.YOUR-PLUGIN-CODE: |-
     resources.limits.cpu: "1000"
     resources.limits.memory: "2000"
 ```
 ### Method 2: Mapped Profile
-1. Create the parameter profile in the `OperatorConfigMap` of the data profile at `data/entando.profile.YOUR-PROFILE-NAME`.
+1. Create the resource parameter profile in the `OperatorConfigMap` of the data profile at `data/entando.profile.YOUR-PROFILE-NAME`, e.g.:
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/devops/plugin-configuration.md:51]: this looks strange with the e.g. here, maybe put down by the yaml example

File: vuepress/docs/next/tutorials/devops/plugin-configuration.md [hunk 3]
```diff
@@ -72,11 +61,11 @@ data:
 ```yaml
 data:
   entando.plugins.profileMapping: |-
-    your-plugin-id: YOUR-PROFILE-NAME
+    YOUR-PLUGIN-CODE: YOUR-PROFILE-NAME
 ```
 
 ### Method 3: Default Profile
-1. Add the profile to the `OperatorConfigMap` in the data section at `data/entando.profile.YOUR-PROFILE-NAME`.
+1. Add the resource parameter profile to the `OperatorConfigMap` in the data section at `data/entando.profile.YOUR-PROFILE-NAME`, e.g.:
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/devops/plugin-configuration.md:68]: same here, with the e.g. It just seems to be left hanging

File: vuepress/docs/next/tutorials/devops/plugin-environment-variables.md [hunk 4]
```diff
@@ -2,60 +2,54 @@
...
-## Add Environment Variables to the Plugin Descriptor File
-1. Determine YOUR-BUNDLE-ID. You can do this from your project directory using this command:
-```sh
-ent prj get-bundle-id
-```
-Alternatively, supply the full bundle URL (including the .git suffix):
+## Add Environment Variables to the Microservice
+1. To determine YOUR-BUNDLE-ID, run the following command from the root project directory. Supply the full bundle URL (including the .git suffix).
 ```sh
 ent ecr get-bundle-id https://github.com/YOUR-ACCOUNT/YOUR-PLUGIN-BUNDLE.git
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/devops/plugin-environment-variables.md:21]: maybe I'm confused, should this be docker? 

And it was not a good idea originally to call this a PLUGIN-Bundle, should change the name

File: vuepress/docs/next/tutorials/devops/plugin-environment-variables.md [hunk 5]
```diff
@@ -2,60 +2,54 @@
...
-environmentVariables:
-  - name: SIMPLE_VAR
-    value: mySimpleValue
-  - name: SECRET_VAR
-    valueFrom:
-      secretKeyRef:
-        name: YOUR-BUNDLE-ID-my-secret
-        key: mySecretKey
+
+3. Insert the following `env` section into the microservice in `entando.json`, remembering to replace `YOUR-BUNDLE-ID`. By convention, environment variables are all caps and K8s resource names are hyphenated.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/devops/plugin-environment-variables.md:29]: add the or your >> the entando.json, ...

File: vuepress/docs/next/tutorials/devops/plugin-environment-variables.md [hunk 6]
```diff
@@ -2,60 +2,54 @@
...
 
-1. When the installation is complete, open a terminal into the pod created by the plugin. `YOUR-PLUGIN-POD-NAME` will begin with `pn-` plus `YOUR-BUNDLE-ID` from above. 
+1. When the installation is complete, shell into the pod created by the plugin, where `YOUR-PLUGIN-POD-NAME` begins with the prefix `pn-` followed by `YOUR-BUNDLE-ID`:
 ```sh
 kubectl exec -it YOUR-PLUGIN-POD-NAME -- /bin/bash
-````
+```
 
-2. Check for the environment variables using the command `env` or this command:
+2. Confirm the environment variables are present:
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/devops/plugin-environment-variables.md:52]: Confirm that the environment...

File: vuepress/docs/next/tutorials/devops/plugin-configuration.md (hunk 2 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/devops/plugin-configuration.md:51]: will remove; "example" not needed

File: vuepress/docs/next/tutorials/devops/plugin-configuration.md (hunk 1 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/devops/plugin-configuration.md:38]: good catch - didn't copy over the full command

File: vuepress/docs/next/tutorials/devops/plugin-environment-variables.md (hunk 4 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/devops/plugin-environment-variables.md:21]: good catch - copy/paste error

agree

File: vuepress/docs/next/tutorials/devops/plugin-environment-variables.md (hunk 5 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/devops/plugin-environment-variables.md:29]: not needed

File: vuepress/docs/next/tutorials/devops/plugin-environment-variables.md (hunk 6 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/devops/plugin-environment-variables.md:52]: not needed

**Comment by nshaw** [comment]
@Lyd1aCla1r3 did you also test the profile and env variables tutorials with these changes? Just checking if we need to run that test still.

**Comment by Lyd1aCla1r3** [comment]
@nshaw i did not. i got as far as working through your scratch pad instructions. i have a bunch of questions about where to go from there; some i can probably answer myself just thinking things through, but it might be better to just cut to the chase next standup. i'll see what i can piece together in the meantime

File: vuepress/docs/next/tutorials/devops/plugin-environment-variables.md (hunk 5 above)
Comment (jyunmitch) [reply vuepress/docs/next/tutorials/devops/plugin-environment-variables.md:29]: add something to the bag, add an apple into the recipe, insert something into the iphone, >> thinking they all need an article. Not completely sure but entando.json is like a toyota, there can be many different ones and you wouldn't say put something into toyota??

Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/devops/plugin-environment-variables.md:29]: Screen Shot 2022-10-04 at 10 15 56 AM

Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/devops/plugin-environment-variables.md:29]: https://english.stackexchange.com/questions/399258/should-i-use-the-before-file-names

File: vuepress/docs/next/tutorials/devops/plugin-environment-variables.md [hunk 7]
```diff
@@ -2,60 +2,56 @@
...
+This tutorial will demonstrate both of these options. The microservice will receive one environment value directly as plain text in the pod YAML and the other indirectly through a referenced Secret.
 
 ## Prerequisites
-* [A project bundle](../create/ms/generate-microservices-and-micro-frontends.md) using the latest Entando Blueprint. 
+* [A bundle project using the latest Entando Blueprint](../create/ms/generate-microservices-and-micro-frontends.md) 
 
-## Add Environment Variables to the Plugin Descriptor File
-1. Determine YOUR-BUNDLE-ID. You can do this from your project directory using this command:
+## Add Environment Variables to the Microservice
+1. To determine YOUR-BUNDLE-ID, run the following command from the root project directory. Supply the full bundle URL, remembering to update the placeholders.
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/devops/plugin-environment-variables.md:19]: Minor, no need to operate from "the root project directory" if you supply the URL.

**Review by jyunmitch — APPROVED** [review]

**Pull request merged**